docs/source/release-notes.rst

```diff
@@ -3,6 +3,21 @@ FiftyOne Release Notes
 
 .. default-role:: code
 
+FiftyOne 0.6.0
+--------------
+*Released October 1, 2020*
+
+App
+^^^
+- Added support for visualizing video datasets
+
+Core
+^^^^
+- Added support for video datasets
```

Comment:
I am expanding this a bit

Reply:
Sounds good, I don't currently plan on making more changes (after https://github.com/voxel51/fiftyone/pull/571/commits/594338ab223e1ab59fd481a0ae649812d0d6c8fc)